docs/design/devices.md

```diff
@@ -0,0 +1,34 @@
+# K8s support "/sys/fs/cgroup/devices"
```

Comment:
device cgroups is one of the access control mechanisms. I'd like to keep that a kubelet implementation detail. This PR and this proposal should center around GPU support.